src/js/electron/brim.ts

```diff
@@ -20,21 +20,21 @@ import isDev from "./isDev"
 import tron, {Session} from "./tron"
 import formatSessionState from "./tron/formatSessionState"
 import {sessionStateFile} from "./tron/session"
-import windowManager, {$WindowManager} from "./tron/windowManager"
+import {WindowManager} from "./tron/window-manager"
 
 type QuitOpts = {
   saveSession?: boolean
 }
 
 type BrimArgs = {
-  windows?: $WindowManager
+  windows?: WindowManager
   store?: Store
   session?: Session
   lake?: Lake
 }
 
 export class BrimMain {
-  readonly windows: $WindowManager
+  readonly windows: WindowManager
```

Comment:
Maybe we should name this member `winManager` to be more consistent its value type, sort of like we do in the tests?

Reply:
Totally. I've gone back and forth on that. In the end, I like the readability of `brim.windows` more than `brim.winManager` when using the brim object.